client/Components/GuestDropdown.jsx

```diff
@@ -101,7 +101,7 @@ class GuestDropdown extends React.Component {
     const { adults, children, infants } = this.state;
     return (
       <div className={styles.dropdown_outerwrap}>
-        <div className={styles.dropdown_container}>
+        <div className={styles.dropdown_}>
```

Comment:
I don't see a dropdown_ class within the GuestDropdown.css file on master. Did you mean to use a different style here?

Reply:
I think this is an editing problem. I'll fix this.